webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -146,7 +147,11 @@ const HostDetails = ({
         >
           {tabs &&
             tabs.map(tab => (
-              <Tab key={tab} eventKey={tab} title={tab}>
+              <Tab
+                key={tab}
+                eventKey={tab}
+                title={TranslateMapper.getTranslatedString(tab)}
```

Comment:
Could there be cases when we do not want to translate the string? I am thinking about 'Ansible' as an example.

Reply:
I can add a fallback 
```js
TranslateMapper.getTranslatedString(tab) || tab
```

Reply:
> Could there be cases when we do not want to translate the string? I am thinking about 'Ansible' as an example.
> Edit: We could probably register untranslated strings if we do not want to translate.

Yes we can register the string as `N_("String")` to not translate it.

Reply:
@upadhyeammit, this `_N()` only works on strings on the server, no? I haven't found equivalent `N_()` in Jed.